src/Symfony/Component/HttpFoundation/Tests/ResponseHeaderBagTest.php

```diff
@@ -18,6 +18,18 @@
 
 class ResponseHeaderBagTest extends \PHPUnit_Framework_TestCase
 {
+    public function setUp()
+    {
+        \Symfony\Component\HttpFoundation\Tests\enable_clock_mock();
```

Comment:
the namespace prefix should be removed since it's the current namespace of the class